src/fmeval/eval_algorithms/faithfulness.py

```diff
@@ -111,24 +112,27 @@ def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
         """
         verdict_output = record[RAW_VERDICTS]
         statements = record[STATEMENTS]
-        record[self.output_key] = self._get_score(verdict_output, statements)
+        record[self.output_key], error = self._get_score(verdict_output, statements)
+        if error is not None:
```

Comment:
Nit (non-blocking): It's more pythonic to just use "if error" since we know that error will either be `None` or a non-empty string.